optuna_dashboard/_storage_url.py

```diff
@@ -58,11 +59,22 @@ def get_storage(
     return guess_storage_from_url(storage)
 
 
+def _has_sqlite_suffix(storage_url: str) -> bool:
+    storage_path = Path(storage_url)
+    SQLITE_EXTENSIONS = (".db", ".sqlite3")
```

Comment:
Actually, Optuna provides its own file storage implementation that may have `.db` suffix.

```python
storage = optuna.storages.JournalStorage(
    optuna.storages.JournalFileStorage("./optuna-journal.db"),
)
```

So Optuna Dashboard could not recognize the storage type from the file extension. Could you check the file header to determine which storages are given, as our browser-only version does?

https://github.com/optuna/optuna-dashboard/blob/9840f2ec03a532f17463b6e640dbd19bcf01d53a/standalone_app/src/components/StorageLoader.tsx#L36-L40

Reply:
Thanks for pointing that out! In the docs i could only find examples with `.log` for the JournalStorage

Reply:
I have changed it as you suggested